magento2php.properties

```diff
@@ -0,0 +1,15 @@
+category=External Analysers
+description=Adds rules for Magento2 PHP development
+homepageUrl=https://github.com/rostilos/sonarqube-magento2-rules
+archivedVersions=
+publicVersions=1.3.0
+
+defaults.mavenGroupId=org.rostilos.sonarrules
+defaults.mavenArtifactId=sonar-magento2-rules
+
+1.3.0.description=PHP Custom Rules for Magento 2 CMS
+1.3.0.sqs=[10.6,24.12]
+1.3.0.sqcb=[24.12,24.12]
```

Comment:
I've just tested this on 25.5 (haven't gotten around to downloading 25.6 yet). You should set this end range to `LATEST`, like so
`1.3.0.sqcb=[24.12,LATEST]`